packages/shared/src/features/shortcuts/hooks/useShortcutLinks.ts

```diff
@@ -132,6 +116,12 @@ export default function useShortcutLinks(): UseShortcutLinks {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [isManual, hasCustomLinks]);
 
+  useEffect(() => {
+    if (hasCustomLinks) {
+      setIsManual(true);
+    }
+  }, [hasCustomLinks, setIsManual]);
```

Comment:
I think there is a good chance we can have this removed by moving the state management of both variables closer to each other. Just optional, non-blocking.

Reply:
I moved this into the context provider so it's closer.